src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/virtual_ips_linux.py

```diff
@@ -95,8 +95,6 @@ async def check_states(self, local, remote):
             errors.append('There are no failover interfaces')
 
         for name in interfaces:
-            if name not in local[0] and name in remote[0]:
-                errors.append(f'Interface "{name}" is MASTER on standby node')
```

Comment:
Could you elaborate? The logic looks right to me, is that a different behavior with keepalived?

Reply:
Using VRRP, there is no "MASTER" or "BACKUP" keyword that gets placed on the interface. Either the VIP exists on the interface or it does not.

Since this method has no notion of which node is MASTER/BACKUP, it will always return as error.

Reply:
Ah, makes sense, thanks